.storybook/.babelrc

```diff
@@ -4,5 +4,6 @@
     ["@emotion/babel-preset-css-prop"],
     ["@babel/preset-react", { "runtime": "automatic", "importSource": "@emotion/react" }],
     ["@babel/preset-typescript"]
-  ]
+  ],
+  "plugins": ["@babel/proposal-class-properties", "@babel/proposal-object-rest-spread"]
```

Comment:
For some reason this is needed to do `private foo: ...` in classes; really thought the `preset-typescript` would take care of that... in retrospect, maybe our tsconfig output is "too new" and leaving the syntax instead of decompiling it down. But eh, probably fine...